docs/quickstart-overview.md

```diff
@@ -1,35 +1,48 @@
-# Quickstart Guide for Percona XtraBackup
+# Overview
 
-Percona XtraBackup (PXB) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+Percona XtraBackup (PXB) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems.
 
-## Install Percona XtraBackup:
+To start with Percona XtraBackup quickly, we recommend using Docker and this Quickstart guide focuses on this installation method. You can explore alternative installation options in the [Install](installation.md) section.
 
-You can install Percona XtraBackup using different methods:
+### Why to start working with Percona XtraBackup in a Docker container
 
-* [Use the Percona Repositories](installation.md)
+With Docker, you can build, deploy, run, update, and manage containers, which isolate applications from the host system. Docker containers are built from Docker images, which are snapshots of the configuration needed to run an applications, such as the code, and libraries. You can run Percona XtraBackup in a Docker container without installing the product.
```

Comment:
The paragraph is confusing. Start talking about containers, then images and then back to containers.

Maybe it would flow better with:
Images
containers and then Percona XtraBackup (why did you add the PXB abbreviation if you are not using it?)
The last sentence may be something like:
A Docker container lets you work with Percona XtraBackup without installing the product on a local drive.

Reply:
fixed